packages/components/src/components/editor/condition-builder/conditionBuilder.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  ConditionBuilderProvider,
+  type LogicalOperator,
+  type LogicOperator,
+  type TypeOperator
+} from '@/components/editor/condition-builder/ConditionBuilderContext';
+import { ConditionBuilder, type ConditionMode } from '@/components/editor/condition-builder/ConditionBuilder';
+import type { Condition } from '@/components/editor/condition-builder/Condition';
+import type { ConditionGroup } from '@/components/editor/condition-builder/ConditionGroup';
+import { useState } from 'react';
+
+const meta: Meta<typeof ConditionBuilder> = {
+  title: 'Editor/ConditionBuilder',
+  component: ConditionBuilder
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ConditionBuilder>;
+
+export const Default: Story = {
+  render: () => {
+    const [condition, setCondition] = useState<string>();
+    return (
+      <>
+        <ConditionBuilderProvider
+          generateConditionString={GenerateConditionString}
+          logicalOperatorOptions={LogicalOperatorOptions}
+          typeOptions={TypeOptions}
+        >
+          <ConditionBuilder onChange={setCondition} />
+        </ConditionBuilderProvider>
+        <pre
+          style={{ border: 'var(--basic-border)', maxHeight: '80px', padding: 'var(--input-padding)', borderRadius: 'var(--border-r2)' }}
+        >
+          {condition}
+        </pre>
+      </>
+    );
+  }
+};
+
+export const TypeOptions: TypeOperator = {
```

Comment:
this is an object and should not start uppercase
```suggestion
export const typeOptions: TypeOperator = {
```

Reply:
I capitalised these objects because otherwise I got an ESLint error: https://jenkins.ivyteam.io/job/ui-components/job/add-condition-builder-to-browser/4/eslint/

Reply:
Ahh because an export in a story is expected as a story in which you then can see in the storybook ui. 
Ok then best define this things in an additional file (like it is done in the table stories I think).
As long as you don't export it in the toplevel index, it will not be included in the bundle output